mypy/newsemanal/semanal.py

```diff
@@ -1293,7 +1264,7 @@ def configure_base_classes(self,
 
         for base, base_expr in bases:
             if isinstance(base, TupleType):
-                if info.tuple_type:
+                if info.tuple_type and info.tuple_type != base:
```

Comment:
Is there any case where `info.tuple_type` and `base` would be equal? I'd expect that at least the fallbacks would be different. Also, we usually use `is_same_type` instead of equality.